mllib/src/test/scala/org/apache/spark/ml/clustering/BisectingKMeansSuite.scala

```diff
@@ -127,6 +128,7 @@ class BisectingKMeansSuite
     assert(clusterSizes.length === k)
     assert(clusterSizes.sum === numRows)
     assert(clusterSizes.forall(_ >= 0))
+    assert(summary.numIter == 2)
```

Comment:
Would be nice to see a test where its not maxIter value being copied over

Reply:
In `KMeansSuite` the value is not `maxIter` (it performs only 1 iteration in that case). In `BisectingKMeans` `numIter` is always `maxIter` since we are always performing `maxIter` (see https://github.com/apache/spark/blob/b6f837c9d3cb0f76f0a52df37e34aea8944f6867/mllib/src/main/scala/org/apache/spark/mllib/clustering/BisectingKMeans.scala#L192).

Does it answer to your comment?